changes/5994-mgorny.md

```diff
@@ -0,0 +1,2 @@
+Make `tests/test_deprecated.py::test_extra_used_as_enum` use
```

Comment:
No need for this file for tiny changes like this.

Reply:
Removed.